meiga/handlers.py

```diff
@@ -0,0 +1,30 @@
+from typing import Callable, Iterable
+
+from meiga.misc import get_args_list
+
+
+class Handler:
+    def __init__(self, func: Callable[..., None], args: Iterable = None):
+        self.func = func
+        self.args = args
+
+    def execute(self, result) -> None:
```

Comment:
Here can we type the result parameter as `Result`?

Reply:
Yes, we should, but it is a circular dependency (handler is called by Result). I tried using quotes but didn't work it!